File: soal-3.js
(() => {
  let arr = [];

  for (let i = 0; i < 10; i++) {
    arr[i] = Number(prompt(`Masukan angka ke-${i + 1} : `));
  }

  let min = arr.reduce((i, r) => (i = Math.min(i, r)), 0);

  console.log(`Angka terkecil adalah : ${min}`);
})();

File: soal-3.py
def find_min_num():
    arr = []
    
    for i in range(0, 10):
        arr.insert(i, int(input(f"Masukan angka ke-{i+1} : ")))
        
    return min(arr)

print(f"\nAngka terkecil adalah : {find_min_num()}")